### DISCUSSION TITLE
Taro 如何实现 share-element 过渡动画

### DISCUSSION THREAD

**Discussion by GiantappMan** [NervJS/taro#10523] [Q & A]
想实现原生的hero效果，不知道taro应该怎么做。有大佬指点一下吗
https://developers.weixin.qq.com/miniprogram/dev/component/share-element.html

**Comment by GiantappMan** [comment]
![TfgVE7jjYU](https://user-images.githubusercontent.com/80653/139292196-a9303021-db03-4683-9df0-f70ae9356efb.gif)

**Comment by Chen-jj** [comment]
`<ShareElement>` 这个组件还没在 Taro 里补全，可以用 `@tarojs/plugin-inject` 先注入试试能不能跑

**Reply by GiantappMan** [reply]
> @tarojs/plugin-inject

有vue编写的示例项目吗。文档看不太懂的样子
